src/PIL/ImageGrab.py

```diff
@@ -22,15 +22,20 @@
 import subprocess
 import sys
 import tempfile
+from typing import TYPE_CHECKING
 
 from . import Image
 
+if TYPE_CHECKING:
+    from . import ImageWin
+
 
 def grab(
     bbox: tuple[int, int, int, int] | None = None,
     include_layered_windows: bool = False,
     all_screens: bool = False,
     xdisplay: str | None = None,
+    handle: int | ImageWin.HWND | None = None,
```

Comment:
```suggestion
    window: int | ImageWin.HWND | None = None,
```
or
```suggestion
    window_handle: int | ImageWin.HWND | None = None,
```
seems clearer.

Reply:
Ok, I've chosen `window`.